lib/passport-verify.ts

```diff
@@ -36,20 +36,29 @@
 import { createStrategy } from './passport-verify-strategy'
 import {
   TranslatedMatchingResponseBody,
+  TranslatedIdentityResponseBody,
   Attributes,
+  IdentityAttributes,
   VerifiableAttribute,
+  VerifiableIdentityAttribute,
   Address,
+  IdentityAddress,
   Scenario
 } from './verify-service-provider-api/translated-response-body'
 
-import { createResponseHandler } from './create-response-handler'
+import { createResponseHandler, createIdentityResponseHandler } from './create-response-handler'
 
 export {
   createStrategy,
   createResponseHandler,
+  createIdentityResponseHandler,
   TranslatedMatchingResponseBody,
+  TranslatedIdentityResponseBody,
   Attributes,
+  IdentityAttributes,
   VerifiableAttribute,
+  VerifiableIdentityAttribute,
   Address,
+  IdentityAddress,
   Scenario
```

Comment:
I think this illustrates pretty well how horrible everything has become!

Reply:
Yep, that's the please. We want to apply that naming convention in ocde wherever the VSP (or clients) is running standalone.

When the VSP is acting on its own: use Identity as a prefix for things
When the MSA is acting with the VSP: use Matching as a prefix for things

Reply:
For our internal consumption, I think it's decent.  I do think it's a huge shame that we've got (for instance) Address versus IdentityAddress in this API though; I don't think that's going to be at all intuitive for RP developers.  If we're stuck with the old names for legacy Passport Verify classes though, so be it.